packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -542,9 +545,30 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
   public async startCamera(options?: VideoStreamOptions): Promise<void> {
     return await this.asyncTeeErrorToEventEmitter(async () => {
       if (!isCameraOn(this.getState())) {
-        await this.handlers.onToggleCamera(options);
+        // First kick off the effect on the local device before starting the camera in the call.
+        // This prevents the effect not being applied for a brief moment when the camera is started.
         /* @conditional-compile-remove(video-background-effects) */
-        await startSelectedVideoEffect(this);
+        {
+          const selectedEffect = this.getState().selectedVideoBackgroundEffect;
+          const videoDeviceInfo = this.getState().devices.selectedCamera || this.getState().devices.cameras[0];
+          if (selectedEffect && videoDeviceInfo) {
+            const stream = new SDKLocalVideoStream(videoDeviceInfo);
```

Comment:
Nice this a neat fix.
Non blocking question: 
Creating a new SDKLocalVideoStream doesn't "turn on" the camera does it, not until we onToggleCamera?
Confirming cause I don't know if there would be any concerns with getting video stream prior to actually turning it on. Would be less of a concern if we had a pre local preview view that contoso can view before actually turning on the camera.

Reply:
It doesn't, but only because of the calling sdk's implementation (it waits until createView is called). Really we should assume it _does_ turn on and use the camera because we are acquiring a stream. In this case its OK to do so because we are calling this on an onStartCamera function so camera access is expected